src/git-lfs/install.sh

```diff
@@ -15,10 +15,6 @@ GIT_LFS_ARCHIVE_GPG_KEY_URI="https://packagecloud.io/github/git-lfs/gpgkey"
 GIT_LFS_ARCHIVE_ARCHITECTURES="amd64 arm64"
 GIT_LFS_ARCHIVE_VERSION_CODENAMES="stretch buster bullseye bionic focal jammy"
```

Comment:
Can we bump patch versions for the updated Features? thanks! 

https://github.com/devcontainers/features/blob/a10c59fbee9fff661d3310f58e95dba6f0056bdb/src/git-lfs/devcontainer-feature.json#L3